src/service/navigation.js

```diff
@@ -310,7 +310,8 @@ export class Navigation {
     if (e.defaultPrevented) {
       return;
     }
-    const target = closestByTag(dev().assertElement(e.target), 'A');
+    const target =
+    closestAncestorElementByTag(dev().assertElement(e.target), 'A');
```

Comment:
Nit: This indentation is wrong.

```js
const element = dev().assertElement(e.target);
const target = closestAncestorElementByTag(element, 'A');
```

Check out the Google JS style guide (we don't follow this exactly): https://google.github.io/styleguide/jsguide.html